### DISCUSSION TITLE
Maps of drives, charges, etc. initially display at World scale

### DISCUSSION THREAD

**Discussion by ampjunkie** [teslamate-org/teslamate#4068] [General]
After updating to v1.30, I see drive maps initially displayed at the World scale, and I have to zoom in to see the drive. This happens with charge maps also and anytime that a map is displayed (except for the map on the front page where the car is located).   Is there some setting that I am missing or do others also see this issue?

**Comment by cwanja** [comment]
Please see #4029. Grafana bug. Will be resolved with a future Grafana update.

**Reply by ampjunkie** [reply]
OK -- great. Thanks!

**Reply by cwanja** [reply]
#4055

**Comment by ampjunkie** [comment]
Thanks for fixing this temporarily in 1.30.1. Much appreciated!
